cola/git.py

```diff
@@ -109,10 +111,13 @@ def __init__(self, git_dir=None, git_file=None, worktree=None, common_dir=None):
         self.common_dir = common_dir
 
     def get(self, path):
+        separator = ":"
+        if sys.platform == "win32":
+            separator = ";"
```

Comment:
Thanks for getting these fixed up. We have a `utils.is_win32()` helper function that checks a few other predicates, and is better for grepping. Please use that function whenever special-casing windows.

TIny sug, but please prefer `'` single-quotes for strings.

todo: I should probably update our `garden fmt` code formatting setup to use `blonde` instead of  `black` so that we can enforce the preference for single-quotes.

Reply:
If code linting (and style enforcement) could come in the form of a pre-commit hook then it will happen automatically.

If you'd like, I'll create a separate PR to introduce a preliminary pre-commit config for trailing whitespace and end-of-file newlines. You can then extend that to use black or isort or blonde or whatever other tools are desirable. 😁

Reply:
I think I've incorporated your feedback correctly; if you spot anything else please let me know!

Reply:
Even better than using `utils.is_win32()` to switch between separators would be to use Python's built-in `os.pathsep` variable which is defined to be the correct separator depending on the platform.

Reply:
@living180 Great callout! I'll switch to that.

Reply:
I've add a commit that switches to `os.pathsep`. Thanks for pointing this out, @living180!